testing/kuttl/e2e/major-upgrade-missing-image/17--check-data-and-version.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
🔧  rename file -- not checking data, just the version, right?

Reply:
Yep, that was a copy/paste thing. Updating.